vsintegration/src/FSharp.Editor/Navigation/GoToDefinitionService.fs

```diff
@@ -296,12 +296,13 @@ type internal FSharpGoToDefinitionService
                         match signatureMetadataService.TryFindMetadataRange (originDocument, ast, symbol) with
                         | Some a -> a
                         | _ -> failwith "unexpected exception in goto definition service"
-                    let sigSourceText = sigDocument.GetTextAsync cancellationToken |> Async.RunTaskSynchronously
+                    let sigSourceText = sigDocument.GetTextAsync() |> Async.RunTaskSynchronously
```

Comment:
this was accidentally removed during debugging this stuff... Thanks for noticing